File: SportEventApplication/www/app/interfaces/sport-event.ts
import { Sport } from './sport';
import { Account } from './account';
import { Address } from './address';

export interface SportEvent {
    Id : number;
    Title : string;
    Date : string;
    Description : string;
    Category: Sport;
    Owner : Account;
    Adress : Address;
    ImagePath : String;
    ImageBase64 : String;
} 

File: SportEventApplication/SportEventApp.Data/Models/SportEvent.cs
using System;
using System.Collections.Generic;

namespace SportEventApp.Data.Models
{
    public class SportEvent : BaseModel
    {
        public string Title { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public int CategoryId { get; set; }
        public virtual Sport Category { get; set; }
        public int OwnerId { get; set; }
        public virtual Account Owner { get; set; }
        public virtual int AdressId { get; set; }
        public virtual Adress Adress {get;set;}
        public virtual ICollection<Account> Attendees { get; set; }
        public string ImagePath { get; set; }

        public SportEvent()
        {
            Attendees = new HashSet<Account>();
        }  
    }
}
